dts/arm/st/f1/stm32f1.dtsi

```diff
@@ -362,6 +362,22 @@
 		v25 = <1430>;
 		ntc;
 	};
+
+	smbus1: smbus1 {
+		compatible = "st,stm32-smbus";
+		status = "disabled";
```

Comment:
nitpick :) Should `disabled` be the last one for easy reading?

Reply:
Skipping through the rest of the file I conclude this is a handled a little bit inconsistent. But I agree that having this line as the last one is better, and I should be consistent at least between smbus1 and smbus2. Will change it.

Reply:
Fixed it and also check the other files again and fixed other similar issues.